_dev/src/components/product-feed/settings/target-countries/target-countries.vue

```diff
@@ -201,6 +244,14 @@ export default {
         name: 'targetCountries', data: countryCode,
       });
     },
+    refreshComponent() {
+      this.$store.dispatch('productFeed/GET_SAVED_ADDITIONAL_SHIPPING_SETTINGS');
+    },
+  },
+  mounted() {
+    this.$store.dispatch('productFeed/GET_SAVED_ADDITIONAL_SHIPPING_SETTINGS').then(() => {
+      this.loadingCountries = false;
+    });
```

Comment:
In case of error, the loadingCountries remains true. Is there some error management to add ?